docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -698,70 +676,64 @@ Waiting for 1 pods to be ready...
 
-Use `kubectl rollout history` to view a history or previous configurations.
+Use the `kubectl rollout history` command to view a history or previous configurations.
 
 ```shell
 kubectl rollout history sts/zk
+
 statefulsets "zk"
 REVISION
 1
 2
 ```
-Use `kubectl rollout undo` to roll back the modification.
+Use the `kubectl rollout undo` command to roll back the modification.
 
 ```shell
 kubectl rollout undo sts/zk
+
 statefulset "zk" rolled back
 ```
 
-### Handling Process Failure 
+### Handling Process Failure
 
-
-[Restart Policies](/docs/user-guide/pod-states/#restartpolicy) control how 
+[Restart Policies](/docs/user-guide/pod-states/#restartpolicy) control how
 Kubernetes handles process failures for the entry point of the container in a Pod.
-For Pods in a StatefulSet, the only appropriate RestartPolicy is Always, and this
-is the default value. For stateful applications you should **never** override 
+For Pods in a `StatefulSet`, the only appropriate `RestartPolicy` is Always, and this
+is the default value. For stateful applications you should **never** override
 the default policy.
 
-
-Examine the process tree for the ZooKeeper server running in the `zk-0` Pod.
+Use the following command to examine the process tree for the ZooKeeper server running in the `zk-0` Pod.
 
 ```shell
 kubectl exec zk-0 -- ps -ef
 ```
 
-The command used as the container's entry point has PID 1, and 
+The command used as the container's entry point has PID 1, and
 the ZooKeeper process, a child of the entry point, has PID 23.
 
 
-```
+```shell
 UID        PID  PPID  C STIME TTY          TIME CMD
 zookeep+     1     0  0 15:03 ?        00:00:00 sh -c zkGenConfig.sh && zkServer.sh start-foreground
 zookeep+    27     1  0 15:03 ?        00:00:03 /usr/lib/jvm/java-8-openjdk-amd64/bin/java -Dzookeeper.log.dir=/var/log/zookeeper -Dzookeeper.root.logger=INFO,CONSOLE -cp /usr/bin/../build/classes:/usr/bin/../build/lib/*.jar:/usr/bin/../share/zookeeper/zookeeper-3.4.9.jar:/usr/bin/../share/zookeeper/slf4j-log4j12-1.6.1.jar:/usr/bin/../share/zookeeper/slf4j-api-1.6.1.jar:/usr/bin/../share/zookeeper/netty-3.10.5.Final.jar:/usr/bin/../share/zookeeper/log4j-1.2.16.jar:/usr/bin/../share/zookeeper/jline-0.9.94.jar:/usr/bin/../src/java/lib/*.jar:/usr/bin/../etc/zookeeper: -Xmx2G -Xms2G -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=false org.apache.zookeeper.server.quorum.QuorumPeerMain /usr/bin/../etc/zookeeper/zoo.cfg
 ```
 
-
-In one terminal watch the Pods in the `zk` StatefulSet.
+In another terminal watch the Pods in the `zk` `StatefulSet` with the following command.
 
 ```shell
 kubectl get pod -w -l app=zk
 ```
 
-
-In another terminal, kill the ZooKeeper process in Pod `zk-0`.
+In another terminal, kill the ZooKeeper process in Pod `zk-0` with the following command.
```

Comment:
s/kill/terminate (I realize that the command itself is `pkill` but we don't need to compound the offense ;) )